apps/client/src/components/atoms/text/Text.component.tsx

```diff
@@ -66,24 +73,49 @@ export function Text({
     className,
     children,
   }: SubTextProps): React.ReactElement => {
+    const content = (
+      <Linkify
+        componentDecorator={(
+          decoratedHref,
+          decoratedText,
+          key,
+        ): ReactElement => (
+          <a
+            target='_blank'
+            rel='noopener noreferrer'
+            href={decoratedHref}
+            key={key}
+            className='text-neutral-40 underline hover:no-underline'>
+            {decoratedText}
+          </a>
+        )}>
+        {children}
+      </Linkify>
+    );
+
     if (size === 'Medium') {
       return (
         <p style={style} className={`${className} font-medium text-indent`}>
-          {children}
+          {content}
         </p>
       );
     } else {
       return (
         <p style={style} className={className}>
-          {children}
+          {content}
         </p>
       );
     }
   };
 
   /** Related text size to the text variant */
   const textSizeValue = TextStyles.TEXT_MAP.get(textVariant);
-  const classNames = (colorClass || 'text-neutral-90') + ' ' + className;
+  const classNames =
+    (colorClass || 'text-neutral-90') +
+    ' ' +
+    className +
+    ' ' +
+    (isAnimatable && 'transition-all duration-300');
```

Comment:
<img width="250" alt="image" src="https://user-images.githubusercontent.com/43986725/220516971-a2ab0a0e-5464-4f8c-91b2-ebec881ddb93.png">

Reply:
Use the ternary operator

Reply:
fixed 7b3db10bf048eb14a05ac33cbb729fc19f70516e